Tests/MapboxDirectionsTests/IntersectionTests.swift

```diff
@@ -16,6 +16,9 @@ class IntersectionTests: XCTestCase {
                 "bearings": [80],
                 "location": [13.426579, 52.508068],
                 "classes": ["toll", "restricted"],
+                "mapbox_streets_v8": [
```

Comment:
Is it possible that there will be multiple street classes? Would it be possible to add test for such case?

Reply:
No, these are classes in the sense of road classification, so they’re mutually exclusive. If Mapbox Streets were to assign multiple classes to a single roadway, it would be a breaking change for Streets anyways.